parsers/FO.py

```diff
@@ -77,9 +79,9 @@ def fetch_production(
     }
     for key, value in obj.items():
         if key == "tiden":
-            data["datetime"] = arrow.get(
-                arrow.get(value).datetime, "Atlantic/Faroe"
-            ).datetime
+            data["datetime"] = datetime.strptime(value, "%Y-%m-%d %H:%M").replace(
+                tzinfo=FO
+            )
```

Comment:
```suggestion
            data["datetime"] = datetime.fromisoformat(value).replace(tzinfo=FO)
```
Since the format returned is based on the ISO format we can use the builtin to parse the date.